### PR TITLE
fix(indemniteLicenciement): update results ui

### PR DESCRIPTION
**Pull request by lionelB**
this pr update the ending screen according to #1364 

image

### CODE REVIEW

File: packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/components/Montant.js [hunk 1]
```diff
@@ -18,32 +18,28 @@ function Montant({ value, primary, ratio = 1 }) {
...
 
 const Bar = styled.div`
-  position: relative;
-  display: inline-flex;
-  align-items: center;
   width: ${({ ratio }) => `calc(50% * ${ratio})`};
-  padding: 1em;
-  background-color: ${({ primary }) =>
-    primary ? colors.blue : colors.lightText};
+  background-color: ${({ theme }) => theme.lightText};
```
Comment (nkrmr) [inline packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/components/Montant.js]: LightText for a background-color ?
There is no better naming ?

**Comment by lionelB** [comment]
poke @nkrmr

File: test.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+.darkGrey {color: #8393a7}
+.lightText {color: #4c5467}
+.darkText {color: #434956}
+.black {color: #0c0c0e}
+.successBackground {color: #e0f2bd}
+.infoBackground {color: #e5f1fe}
+.warningBackground {color: #fee5ad}
+.dangerBackground {color: #f6bcc2}
+.focus {color: #80bdff}
+.errorText {color "#b5011}
```
Comment (nkrmr) [inline test.css]: We have to keep this file here?

Comment (lionelB) [reply test.css]: oh noooooo T_T

**Review by nkrmr — APPROVED** [review]

**Pull request merged**